tests.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'classes/nameable'
+require_relative 'classes/person'
+require_relative 'classes/student'
+require_relative 'classes/teacher'
+require_relative 'classes/decorator'
+require_relative 'classes/capitalize_decorator'
+require_relative 'classes/trimmer_decorator'
+require_relative 'classes/book'
+require_relative 'classes/rental'
+require_relative 'classes/classroom'
+
+person = Teacher.new(22, 'History', 'maximilianus')
+person2 = Teacher.new(24, 'Geography', 'Julian')
+person3 = Teacher.new(32, 'Universal History', 'Hanna')
+# person4 = Student.new(13, 'Dean')
+# person5 = Student.new(16, 'Felicia')
```

Comment:
- [OPTIONAL] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻 Since it's the only thing. I'm gonna approve this PR.